src/Nordea/Components/StepIndicator.elm

```diff
@@ -121,6 +87,7 @@ listStyles =
     [ displayFlex
     , listStyleType none
     , alignItems center
+    , paddingLeft (px 0)
```

Comment:
In PartnerHub, we try to never use px. Doesn't really matter for 0 though 🙂
```suggestion
    , paddingLeft (rem 0)
```

Reply:
ofc! I fix 🙌